### PR TITLE
Correctly pre-bind the createMachine method in MV

### LINKED ISSUE #3227: new-init: can't create a new machine
In the machine view if you open the form to create a new machine or container and click 'Create' there is an error `Uncaught TypeError: Cannot read property 'machines' of null` (for machines) or `Uncaught TypeError: Cannot read property 'addGhost' of undefined` (for containers).

### PR DESCRIPTION
**Pull request by hatched**
Fixes #3227

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

Follow steps in issue and make sure you can now create new machines and add deploy units to them.

File: Makefile [hunk 1]
```diff
@@ -24,7 +24,8 @@ YUI := $(NODE_MODULES)/yui
 BUILT_YUI := $(BUILT_JS_ASSETS)/yui
 BUILT_D3 := $(BUILT_JS_ASSETS)/d3-min.js
 SELENIUM := lib/python2.7/site-packages/selenium-2.47.3-py2.7.egg/selenium/selenium.py
-REACT_ASSETS := $(BUILT_JS_ASSETS)/react-with-addons.js $(BUILT_JS_ASSETS)/react-with-addons.min.js
+# This REACT_ASSETS is simply a canary for the target
+REACT_ASSETS := $(BUILT_JS_ASSETS)/classnames.js
```
Comment (hatched) [inline Makefile]: Driveby, we don't have those react assets any longer so just using another file from the list.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8166/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8167/

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge
